### ISSUE TITLE
Advanced search in database would be useful

### ISSUE THREAD

**Issue by archimedus** [SaschaWillems/VulkanCapsViewer#63]
I need to find a hardware that supports `VK_QUEUE_PROTECTED_BIT`, though it seems there is no such request.

Specialized search in VkQueueFlagBits might be useful, though  I suppose general site search capability might be useful.

**Comment by SaschaWillems** [comment]
That's true, I'm not happy with the limited search capabilities myself and that's something I want to improve. I'm not sure how to easily add some kind of advanced search, but I'm planning to add something that'll add more search options.

**Comment by krys-g** [comment]
I humbly second that request.

More generally, the ability to search for the reported values of a specific field across the device database would be particularly useful.
